src/main/java/uk/gov/justice/record/link/service/UserTransferService.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.record.link.service;
+
+import org.springframework.stereotype.Service;
+import org.thymeleaf.util.StringUtils;
+import uk.gov.justice.record.link.entity.CcmsUser;
+import uk.gov.justice.record.link.entity.LinkedRequest;
+import uk.gov.justice.record.link.entity.Status;
+import uk.gov.justice.record.link.model.UserTransferRequest;
+import uk.gov.justice.record.link.respository.CcmsUserRepository;
+import uk.gov.justice.record.link.respository.LinkedRequestRepository;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Service
+public class UserTransferService {
+
+    private final LinkedRequestRepository linkedRequestRepository;
+    private final CcmsUserRepository ccmsUserRepository;
+
+    public UserTransferService(LinkedRequestRepository linkedRequestRepository, CcmsUserRepository ccmsUserRepository) {
+        this.linkedRequestRepository = linkedRequestRepository;
+        this.ccmsUserRepository = ccmsUserRepository;
+    }
+
+    public LinkedRequest save(final UserTransferRequest userTransferRequest) {
+
+        CcmsUser ccmsUser = CcmsUser.builder().loginId(userTransferRequest.getOldLogin()).build();
+        ccmsUserRepository.save(ccmsUser);
+
```

Comment:
ccms_user will given populated. We don't need to save, but rather get the CcmsUse using findByLoginId